index.html

```diff
@@ -30,15 +30,4 @@
     </main>
     <footer id="footer"><a>Click Here</a></footer>
 </body>
-</html>
```

Comment:
@NahnahAJ  Please use html tag to close the already open `<html>` tag. Kindly put `</html>` tag again in your code.